guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/PullRequestService.java

```diff
@@ -0,0 +1,33 @@
+package org.guvnor.structure.repositories;
+
+import java.util.List;
+
+import org.uberfire.java.nio.base.FileDiff;
+
+public interface PullRequestService {
+
+    PullRequest createPullRequest( PullRequest pullRequest );
+
+    PullRequest acceptPullRequest( String repository,
```

Comment:
Presumably ```id``` is the ```PullRequest.getId()``` and ```repository``` is either ```PullRequest.getTargetRepository()``` or ```PullRequest.getSourceRepository()```? Is passing a ```PullRequest``` parameter better encapsulation? IDK.